.gitignore

```diff
@@ -1,23 +1,7 @@
-dist
-dist-*
-cabal-dev
-*.o
-*.hi
-*.hie
-*.chi
-*.chs.h
-*.dyn_o
-*.dyn_hi
-.hpc
-.hsenv
-.cabal-sandbox/
-cabal.sandbox.config
-*.prof
-*.aux
-*.hp
-*.eventlog
-.stack-work/
-cabal.project.local
-cabal.project.local~
-.HTF/
-.ghc.environment.*
+.stack-work
+db.json
+.DS_Store
```

Comment:
Isn't it something MacOS-specific? I think MacOS users most likely have it in their global `.gitignore`.